i18n/usage/de.txt

```diff
@@ -0,0 +1,22 @@
+{bold}{yellow}Verwendung{/yellow}{/bold}
+
+  {bold}{green}{appname}{/green}{/bold}
+    Zeige ein Menü, um interaktiv einen Workshop auszuwählen.
+  {bold}{green}{appname}{/green} list{/bold}
+    Zeige eine Liste aller Workshops, ein Workshop pro Zeile.
+  {bold}{green}{appname}{/green} select NAME{/bold}
+    Wähle einen Workshop.
+  {bold}{green}{appname}{/green} current{/bold}
+    Zeige den aktuell gewählten Workshop.
+  {bold}{green}{appname}{/green} print{/bold}
+    Gebe die Anweisungen für den aktuell gewählen Workshop aus.
```

Comment:
dann scheint die englische variante an dieser stelle auch nicht ganz eindeutig zu sein?
"Print the instructions for the currently selected workshop." exercise/challenge wäre da auch besser oder?